newIDE/app/src/AssetStore/PrivateAssetPackDialog.js

```diff
@@ -0,0 +1,244 @@
+                    <Trans>by</Trans>{' '}
+                    <Link
+                      onClick={() => setOpenSellerPublicProfileDialog(true)}
+                      href=""
+                    >
+                      {sellerPublicProfile.username || ''}
+                    </Link>
+                  </Text>
+                </Column>
+                <ResponsiveLineStackLayout noColumnMargin noMargin>
+                  <Column
+                    useFullHeight
+                    expand={windowWidth !== 'small'}
+                    noMargin
+                    noOverflowParent
+                  >
+                    <ImagesDisplay
+                      imagesUrls={assetPackDetails.previewImageUrls}
+                      altTextTemplate={`Asset pack ${name} preview image {imageIndex}`}
+                      horizontalOuterMarginToEatOnMobile={8}
+                    />
+                  </Column>
+                  <Column useFullHeight expand noMargin>
+                    <Paper
+                      variant="outlined"
+                      style={{ padding: windowWidth === 'small' ? 20 : 30 }}
+                    >
+                      <Column noMargin>
+                        <LargeSpacer /> {/* To be replaced by prices */}
+                        <Text noMargin>{assetPackDetails.longDescription}</Text>
+                        <LargeSpacer />
+                        <ResponsiveLineStackLayout noMargin noColumnMargin>
+                          <Column noMargin expand>
+                            <Text size="sub-title">
+                              <Trans>Content</Trans>
+                            </Text>
+                            {sortedContentType.map(type => {
+                              if (assetPackDetails.content[type]) {
+                                return (
+                                  <li key={type}>
+                                    <Text displayInlineAsSpan noMargin>
+                                      {assetPackDetails.content[type]}{' '}
+                                      {i18n._(
+                                        contentTypeToMessageDescriptor[type]
+                                      )}
+                                    </Text>
+                                  </li>
+                                );
+                              }
+                              return null;
+                            })}
+                          </Column>
+                          <Column noMargin expand>
+                            <Text size="sub-title">
+                              <Trans>Licensing</Trans>
+                            </Text>
+                            <Line noMargin alignItems="center">
+                              <Mark fontSize="small" />
+                              <Spacer />
+                              <Text displayInlineAsSpan noMargin>
+                                <Trans>Personal projects</Trans>
```

Comment:
I guess those elements should be dependent on the Licensing of the Asset Pack?
Or are all private asset packs under the same license?

Reply:
For the moment, we decided it will be the same licensing for every asset pack